sql/core/pom.xml

```diff
@@ -147,6 +147,12 @@
       <artifactId>mockito-core</artifactId>
       <scope>test</scope>
     </dependency>
+    <!-- RocksDB dependency for Structured Streaming State Store -->
+    <dependency>
+      <groupId>org.rocksdb</groupId>
+      <artifactId>rocksdbjni</artifactId>
+      <version>6.0.1</version>
```

Comment:
Since rocksdb impacts sql/core module, is it fair to move it to the root pom? I see that there are other dependencies whose version is part of module pom. I can make the changes if moving to root pom is the guidelines.

Reply:
I mean store the version purely in root pom and then include it here.
```
<rocksdb.version>6.0.1</rocksdb.version>
```
```
      <groupId>org.rocksdb</groupId>
      <artifactId>rocksdbjni</artifactId>
      <version>${rocksdb.version}</version>
```